homework-21/hw21-task1.py

```diff
@@ -0,0 +1,50 @@
+import json
+
+try:
+    with open('data.json', 'r') as file:
+        data = json.load(file)
+except FileNotFoundError:
+    print("Error: The file 'data.json' does not exist.")
+    exit()
+
+class Employee:
+    def __init__(self, id, name, position, salary):
+        self.id = id
+        self.name = name
+        self.position = position
+        self.salary = salary
+
+class Department:
+    def __init__(self, employees):
+        self.employees = employees
+
+    def average(self):
+        total_salary = sum(emp.salary for emp in self.employees)
```

Comment:
შეგვიძლია ასეთი დამატებებით თავიდან ავირიდოთ შემთხვევები დეპარტამენტებში სადაც არაა თანამშრომელი,ერორების ასარიდებლად.
 
    if not self.employees:
        print("No employees in the department.")
        return 0